pkg/synthetictests/allowedalerts/all.go

```diff
@@ -1,6 +1,31 @@
 package allowedalerts
 
-func AllAlertTests() []AlertTest {
+import (
+	"context"
+
+	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
+	"k8s.io/client-go/rest"
+)
+
+// AllAlertTests returns the list of AlertTests with independent tests instead of a general check.
+// clientConfig may be nil, but the quality of the allowances will be better if it is set.
+// You may choose to send nil to get a list of names for instance.
+func AllAlertTests(ctx context.Context, clientConfig *rest.Config) []AlertTest {
+	var etcdAllowance AlertTestAllowanceCalculator
+	etcdAllowance = defaultAllowances
+
+	// if we have a clientConfig,  use it.
+	if clientConfig != nil {
+		operatorClient, err := operatorv1client.NewForConfig(clientConfig)
+		if err != nil {
+			panic(err)
+		}
+		etcdAllowance, err = NewAllowedWhenEtcdRevisionChange(ctx, operatorClient)
+		if err != nil {
+			panic(err)
```

Comment:
this can take down the whole suite

Reply:
> this can take down the whole suite

Yes.  If we cannot get the state of our etcd operator, why continue?

Reply:
setting up a cluster is expensive, a single network error before running a single test could ruin that. Now I see we only issue a `GET` request which might be retried internally.